R/zcurve.R

```diff
@@ -0,0 +1,31 @@
+#' Create a curve comparing observed Z-scores to the WHO standard.
+#'
+#' @param zscore a numeric vector containing computed zscores
+#' @return a ggplot2 object
+#' @export
+#' @importFrom ggplot2 ggplot aes stat_function geom_density scale_x_continuous labs
+#' @examples
+#' dat <- rnorm(204) + runif(1) # slightly skewed
+#' zcurve(dat)
+zcurve <- function(zscore) {
+
+  stopifnot(is.numeric(zscore))
+  dat <- data.frame(observed = zscore)
+  obs <- quote(observed)
+  ggplot(dat) +
+    geom_density(aes(x = !!obs, color = "observed")) +
```

Comment:
Isn't there `aes_string` for that?

Reply:
Yes there is, but the function is [soft deprecated](https://ggplot2.tidyverse.org/reference/aes_.html#life-cycle), so I would rather go with the recommended version